WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodTableViewController.swift

```diff
@@ -36,7 +36,7 @@ final class SiteStatsPeriodTableViewController: SiteStatsBaseTableViewController
     private let analyticsTracker = BottomScrollAnalyticsTracker()
 
     private lazy var tableHandler: ImmuTableDiffableViewHandler = {
-        return ImmuTableDiffableViewHandler(takeOver: self, with: analyticsTracker)
```

Comment:
Was it removed intentionally?

Reply:
Yes, we don't track scrolling anymore.

Reply:
`BottomScrollAnalyticsTracker` appears to be unused now and can be deleted.

Reply:
There's still some usage of analyticsTracker by jetpackBanner on the WordPress app. I think it all can be removed since we have a much later migration stage enabled but it needs to be checked. However, I'll leave it out of scope, I think all this migration code needs to be reviewed separately.